Missing packages in vcpkg prerequisites page

**What does the current documentation state?**
In "[vcpkg.rst](https://github.com/moja-global/moja_global_docs/blob/master/docs/prerequisites/vcpkg.rst)" it lists the following command for installing packages for `vcpkg`:

```
vcpkg.exe install boost-test:x64-windows boost-program-options:x64-windows boost-log:x64-windows turtle:x64-windows zipper:x64-windows poco:x64-windows libpq:x64-windows gdal:x64-windows sqlite3:x64-windows boost-ublas:x64-windows
```

**Where is this stated?**
In "[vcpkg.rst](https://github.com/moja-global/moja_global_docs/blob/master/docs/prerequisites/vcpkg.rst)"

**Why do you want to improve the statement?**
The current command is missing two packages: `fmt:x64-windows` and `libpqxx:x64-windows`. So if readers complete the prerequisite instructions to install `vcpkg` when they try to compile with CMake they will receive the following error:

```
CMake Error at C:/Development/moja-global/vcpkg/scripts/buildsystems/vcpkg.cmake:288 (_find_package):
  Could not find a package configuration file provided by "fmt" with any of
  the following names:

    fmtConfig.cmake
    fmt-config.cmake
```

**Proposed statement**
The command should be corrected to the following:
```
vcpkg.exe install boost-test:x64-windows boost-program-options:x64-windows boost-log:x64-windows turtle:x64-windows zipper:x64-windows poco:x64-windows libpq:x64-windows gdal:x64-windows sqlite3:x64-windows boost-ublas:x64-windows fmt:x64-windows libpqxx:x64-windows
```

**Additional info**
Installing `vcpkg` is also mentioned in "[windows_installation.rst](https://github.com/moja-global/moja_global_docs/blob/master/docs/DevelopmentSetup/windows_installation.rst)" and "[visual_studio_win_example.rst](https://github.com/moja-global/moja_global_docs/blob/master/docs/DevelopmentSetup/visual_studio_win_example.rst)" but those commands are  both the same and contain `fmt:x64-windows`. Making this correction to "[vcpkg.rst](https://github.com/moja-global/moja_global_docs/blob/master/docs/prerequisites/vcpkg.rst)" will mean all pages have the same `vcpkg` command.

--- a/docs/prerequisites/vcpkg.rst
+++ b/docs/prerequisites/vcpkg.rst
@@ -17,5 +17,5 @@ To build the libraries please follow the following steps:
     bootstrap-vcpkg.bat
 
     # install packages
-    vcpkg.exe install boost-test:x64-windows boost-program-options:x64-windows boost-log:x64-windows turtle:x64-windows zipper:x64-windows poco:x64-windows libpq:x64-windows gdal:x64-windows sqlite3:x64-windows boost-ublas:x64-windows
+    vcpkg.exe install boost-test:x64-windows boost-program-options:x64-windows boost-log:x64-windows turtle:x64-windows zipper:x64-windows poco:x64-windows libpq:x64-windows gdal:x64-windows sqlite3:x64-windows boost-ublas:x64-windows fmt:x64-windows libpqxx:x64-windows
 

